mondoo_dictionary.txt

```diff
@@ -369,6 +370,7 @@ internetgateway
 ipinterface
 ippermission
 iptable
+isp
```

Comment:
Should this be caps?

Reply:
It's from a resource so not in this case